task-driver/integration/main.rs

```diff
@@ -91,10 +77,10 @@ struct CliArgs {
 /// The arguments provided to every integration test
 #[derive(Clone)]
 struct IntegrationTestArgs {
-    /// The address of the darkpool deployed on the Arbitrum devnet
-    darkpool_addr: String,
-    /// The address of a pre-deployed ERC20 for testing
-    erc20_addr: String,
+    /// The address of the first pre-deployed ERC20 for testing
+    erc20_addr_0: String,
+    /// The address of the second pre-deployed ERC20 for testing
+    erc20_addr_1: String,
```

Comment:
nit: let's use consistent style with the rest of this crate: `erc20_addr0` and `erc20_addr1`, same below